### DISCUSSION TITLE
bug: Filters are not kept when user refreshes the page

### DISCUSSION THREAD

**Discussion by sevilayerkan** [acikkaynak/deprem-yardim-frontend#1054] [Ideas]
## Bug Definition
Filters are not kept when user refreshes the page

-- Please add discord contact information
** discord username: @notDepresseDeveloper# 0919 **

## Bug environment
Describe the environment produces the bug. For example:
rc.afetharita.com
afetharita.com

## Describe how you are producing the bug step by step

1. Go to afetharita.com or rc.afetharita.com
2. Select something from filters
3. Refresh the page
4. Bug appears

## Expected Behaviour
Filters have to be cached and user should see previous filter they choose.

## Desktop Information
- Operating System: Windows 10, 11
- Browser: Chrome, Safari

## Mobile Phone Information
- Operating System (with the version): Android, IOS 
- Browser : Chrome, Safari (for IOS), Samsung Browser (Android)

**Comment by ugurrdemirel** [comment]
I think it's a big problem for people who organize aids. These filters should stay when user refresh page. I'm already prepared PR for this but not send because of PR rules.

Someone needs to approve this issue.

>  PR must be created for an issue with approved tag. Otherwise PR will be rejected.

**Comment by usirin** [comment]
We need to save the filter state to url.

**Comment by snturk** [comment]
As a result of #996, there can be a link like below to share with others with filters in it.

`https://afetharita.com/en?lat=38.094574707566274&lng=35.211181640625&zoom=7&lastTime=last1Hour&channel=twitter&reasons=yemek%2Cerzak`

**Comment by handekaratas96** [comment]
A regular user won't understand how to copy the link and share it with someone else. Thus, "share" button should be added to the ui 
selected filter settings should be shared with the "share button" via the shorter version of the url suggested as above
we need to discuss this further

@canerakin111 @nazdezar
